core/src/scss/utilities/variables/core/_grid.scss

```diff
@@ -1,122 +1,226 @@
-// Style guide: Variables.Core.grid-media
+// Style guide: Variables.Core.Grid.su-grid-media
 //
-$grid-media: (
-  'xs': $breakpoint-xs,
-  'sm': $breakpoint-sm,
-  'md': $breakpoint-md,
-  'lg': $breakpoint-lg,
-  'xl': $breakpoint-xl,
-  '2xl': $breakpoint-2xl
+$su-grid-media: (
+  'xs': $su-breakpoint-xs,
+  'sm': $su-breakpoint-sm,
+  'md': $su-breakpoint-md,
+  'lg': $su-breakpoint-lg,
+  'xl': $su-breakpoint-xl,
+  '2xl': $su-breakpoint-2xl
 ) !default;
 
-// $grid-media-only
+//
+// <del>$grid-media</del>
+//
+// <strong>Deprecated. Will be removed in 6.0.0.</strong><br/>
+// Use <code>$su-grid-media</code> instead.
+//
+// Style guide: Variables.Core.Grid.grid-media
+//
+$grid-media: $su-grid-media !default;
+
+
+// $su-su-grid-media-only
 //
 // A sass map of single breakpoint spans keyed with the breakpoint shortcode.
 //
-// Style guide: Variables.Core.grid-screens-only
+// Style guide: Variables.Core.Grid.su-grid-screens-only
 //
-$grid-media-only: (
-  'xs': $breakpoint-xs-only,
-  'sm': $breakpoint-sm-only,
-  'md': $breakpoint-md-only,
-  'lg': $breakpoint-lg-only,
-  'xl': $breakpoint-xl-only,
-  '2xl': $breakpoint-2xl
+$su-grid-media-only: (
+  'xs': $su-breakpoint-xs-only,
+  'sm': $su-breakpoint-sm-only,
+  'md': $su-breakpoint-md-only,
+  'lg': $su-breakpoint-lg-only,
+  'xl': $su-breakpoint-xl-only,
+  '2xl': $su-breakpoint-2xl
 ) !default;
 
-// $responsive-columns-default
+//
+// <del>$grid-media-only</del>
+//
+// <strong>Deprecated. Will be removed in 6.0.0.</strong><br/>
+// Use <code>$su-grid-media<-only</code> instead.
+//
+// Style guide: Variables.Core.Grid.grid-media-only
+//
+$grid-media-only: $su-grid-media-only !default;
+
+
+// $su-responsive-columns-default
 //
 // A sass map of css grid columns keyed with breakpoint media queries. 2 column.
 //
-// Style guide: Variables.Core.responsive-columns-default
+// Style guide: Variables.Core.Grid.su-responsive-columns-default
 //
-$responsive-columns-default: (
-  $breakpoint-xs: 1fr,
-  $breakpoint-sm: 1fr 1fr,
-  $breakpoint-md: 1fr 1fr,
-  $breakpoint-lg: 1fr 1fr,
-  $breakpoint-xl: 1fr 1fr,
-  $breakpoint-2xl: 1fr 1fr
+$su-responsive-columns-default: (
+  $su-breakpoint-xs: 1fr,
+  $su-breakpoint-sm: 1fr 1fr,
+  $su-breakpoint-md: 1fr 1fr,
+  $su-breakpoint-lg: 1fr 1fr,
+  $su-breakpoint-xl: 1fr 1fr,
+  $su-breakpoint-2xl: 1fr 1fr
 ) !default;
 
+//
+// <del>$responsive-columns-default</del>
+//
+// <strong>Deprecated. Will be removed in 6.0.0.</strong><br/>
+// Use <code>$su-responsive-columns-default</code> instead.
+//
+// Style guide: Variables.Core.Grid.responsive-columns-default
+//
+$responsive-columns-default: $su-responsive-columns-default !default;
+
+
 // $responsive-columns-two
 //
 // A sass map of css grid columns keyed with breakpoint media queries. 2 column.
 //
-// Style guide: Variables.Core.responsive-columns-two
+// Style guide: Variables.Core.Grid.responsive-columns-two
+//
+$responsive-columns-two: $su-responsive-columns-default !default;
```

Comment:
Do you mean `$su-responsive-columns-two: $su-responsive-columns-default !default; `here? Also the path in the "Style guide:" line.

Reply:
my mind has gone mushy.

Reply:
There are just so many variables 😄  You did very well, @cjwest !